src/main/java/com/taboola/backstage/services/CampaignsService.java

```diff
@@ -79,6 +79,33 @@
      */
     Campaign update(BackstageAuthentication auth, String accountId, String campaignId, CampaignOperation campaignOperation) throws BackstageAPIUnauthorizedException, BackstageAPIConnectivityException, BackstageAPIRequestException;
 
+    /**
+     * Duplicate campaign entity
```

Comment:
I think worth mentioning the difference between the two, except the difference in the signature.
something like "Duplicate campaign entity without modifying it" or so...

Reply:
Done